modules/moa-installing-prerequisites.adoc

```diff
@@ -24,7 +24,16 @@ This user will have Programmatic access enabled and the AdministratorAccess poli
 ====
 +
 . Install and configure the AWS CLI.
-.. Follow the AWS documentation to install the link:https://aws.amazon.com/cli/[aws-cli] for your operating system.
+.. Follow the AWS documentation to install and configure the link:https://aws.amazon.com/cli/[aws-cli] for your operating system.
++
+[NOTE]
+====
+`moactl` Priority Order of AWS Regions::
+... Option specified on the `moactl` command line +
+... Environment variable +
+... Default region listed in the AWS configuration file +
+====
++
```

Comment:
This is difficult because we're trying to provide helpful pointers on how to use the AWS CLI, without getting too deep into redocumenting what AWS docs already cover.

Here's a proposed update with some links:
`. Install and configure the AWS CLI.
 .. Follow the AWS documentation to link:https://aws.amazon.com/cli/[install and configure the aws-cli] for your operating system.
 .. Optional: Set a default AWS region. `moactl` evaluates regions in the following priority order:
 +
... The region specified when running a `moactl` command with the `--region` flag.
... The region set in the `AWS_DEFAULT_REGION` environment variable. See the link:https://docs.aws.amazon.com/cli/latest/userguide/c    li-configure-envvars.html[AWS documentaiton for more details].
... The default region set in your AWS configuration file. See the link:https://docs.aws.amazon.com/cli/latest/userguide/cli-configu    re-quickstart.html#cli-configure-quickstart-config[AWS documentation for more details].`

Reply:
![image](https://user-images.githubusercontent.com/3738757/96002573-44659500-0e07-11eb-93d9-f046905c98f5.png)